src/pages/guidelines/color/overview.mdx

```diff
@@ -261,54 +289,230 @@ attention or create visual tension.
   </DoDont>
 </DoDontRow>
 
+<br />
+
+### Tokens
+
+Color in Carbon is managed through tokens. Tokens are an abstracted method of
+applying color in a consistent and scalable way. Each color token is assigned a
```

Comment:
Maybe reusable could be added in "consistent, reusable, and scalable" as another benefit of using tokens